pkg/transform/oauth_transform.go

```diff
@@ -70,16 +69,15 @@ func (e OAuthTransform) Extract() (Extraction, error) {
 		return nil, err
 	}
 
-	serializer := k8sjson.NewYAMLSerializer(k8sjson.DefaultMetaFactory, scheme.Scheme, scheme.Scheme)
-	var masterConfig configv1.MasterConfig
-	var extraction OAuthExtraction
-	var htContent, caContent, crtContent, keyContent []byte
-
-	_, _, err = serializer.Decode(content, nil, &masterConfig)
+	var masterConfig *configv1.MasterConfig
+	masterConfig, err = decode.MasterConfig(content)
 	if err != nil {
 		return nil, err
 	}
 
+	var extraction OAuthExtraction
+	var htContent, caContent, crtContent, keyContent []byte
```

Comment:
If you aren't going to use these outside the loop I would simply declare them using the `:=`, see next comment.

Reply:
On line 95 changed it to be `htContent, err := e.Config.Fetch(provider.File)`. Typically when you create a var outside a block you intend to use that variable elsewhere.

Reply:
Yes, I agree, just that I didn't pass on all existing assignments.